### ISSUE TITLE
controls bug - can't bind [TAB] key

### ISSUE THREAD

**Issue by Owyn** [yuzu-emu/yuzu#6420]
you can bind any other button in controls on the keyboard but TAB instead just switches UI elements and won't get binded

**Comment by german77** [comment]
Tab it's reserved for that purpose as it's a navigation key. It might be posible to disable navigation temporarily on mapping

**Comment by adityaruplaha** [comment]
This stems from Citra's original codebase. I remember this was brought up in conversation once, and it was decided to keep this behaviour for accessibility reasons. It probably was also present in the original swkbd implementation.

**Comment by Owyn** [comment]
You can bind [TAB] in Ryujinx controls perfectly fine... just saying

**Comment by soredake** [comment]
Ability to bind tab would be nice to have.

**Comment by octopushugger** [comment]
Just found this because the very first time I'm binding buttons to the keyboard I immediately try to bind escape for pause and tab for map. Kind of insane to me that these keys were ever made this way for any gaming software, especially an emulator. 

Proposed solution: Move current escape and tab actions to General->Hotkeys so users can rebind them as they wish.

**Comment by adityaruplaha** [comment]
As I had said in my earlier comment, Tab and Escape are used by Qt (our UI library) to handle navigation. While it should technically be possible to disable navigation during mapping as @german77 pointed out, this wouldn't be a simple solution. It might involve reworking the entire system. 

Moving the current action to Hotkeys isn't possible, as this is a Qt binding, not one created by yuzu. 

In addition, keyboard-based navigation would have to disabled completely for this to work, because, IIRC, Qt event handlers for navigation keys take first precedence.

TL;DR - This is essentially a Qt limitation. Without major changes, I don't see how it is possible to implement this. 

Requesting this be tagged as a feature request, not a bug.

**Comment by octopushugger** [comment]
Could potentially spawn some sort of input devouring process when binding as a workaround if disabling Qt nav isn't possible without heavy modification.

**Comment by kotisorion** [comment]
I've figured out how to do this.
First find Yuzu\user\config\qt-config.ini
For player 1 controller navigate to the player_0 button map under the [Controls] header (line 368 for me). Find the button you wish to assign tab to and change the code to 16777217.
My personal example below is for the zL key:
player_0_button_zl="engine:keyboard,port:0,pad:0,code:16777217"

**Comment by meikiuwu** [comment]
> I've figured out how to do this. First find Yuzu\user\config\qt-config.ini For player 1 controller navigate to the player_0 button map under the [Controls] header (line 368 for me). Find the button you wish to assign tab to and change the code to 16777217. My personal example below is for the zL key: player_0_button_zl="engine:keyboard,port:0,pad:0,code:16777217"

How'd you figure out the code name?
